files/homeassistant/hacs_version/individual_control.yaml

```diff
@@ -176,102 +197,86 @@ cards:
+              background-color: rgba(0,0,0,0);
+              border: none;
+              --paper-item-icon-color: {% if is_state('sensor.{HA_PRINTER_DEVICE_NAME}_print_status', 'RUNNING') or is_state('sensor.{HA_PRINTER_DEVICE_NAME}_print_status', 'PAUSE') or is_state('sensor.{HA_PRINTER_DEVICE_NAME}_print_status', 'running') or is_state('sensor.{HA_PRINTER_DEVICE_NAME}_print_status', 'pause') %} orange; {% else %} grey; {% endif %}    
+            }
       - type: custom:gap-card
       - type: custom:button-card
         entity: sensor.{HA_PRINTER_DEVICE_NAME}_print_status
         name: |
-          [[[  if (entity.state == 'running') return "Cancel";
-           else if (entity.state =='pause') return "Cancel";
+          [[[  if (entity.state.upper == 'RUNNING') return "Cancel";
+           else if (entity.state.upper =='PAUSE') return "Cancel";
           else return '';  ]]]
         confirmation:
           text: Cancel Print?
         state:
+          - value: RUNNING
+            icon: mdi:cancel
+            color: red
           - value: running
             icon: mdi:cancel
             color: red
+          - value: PAUSE
+            icon: mdi:cancel
+            color: red
           - value: pause
             icon: mdi:cancel
             color: red
+          - value: OFFLINE
+            icon: mdi:void
+            color: grey
           - value: offline
             icon: mdi:void
             color: grey
+          - value: IDLE
+            icon: mdi:void
+            color: grey
           - value: idle
             icon: mdi:void
             color: grey
+          - value: FAILED
+            icon: mdi:void
+            color: grey
           - value: failed
             icon: mdi:void
             color: grey
+          - value: FINISH
+            icon: mdi:void
+            color: grey
           - value: finish
             icon: mdi:void
             color: grey
+          - value: PREPARE
+            icon: mdi:void
+            color: grey
           - value: prepare
             icon: mdi:void
             color: grey
         tap_action:
           action: >
-            [[[ if (entity.state == 'running' || entity.state == 'pause') return
+            [[[ if (entity.state.upper == 'RUNNING' || entity.state.upper == 'PAUSE') return
             "call-service";  else return "none"; ]]]
           service: button.press
           service_data:
             entity_id: button.{HA_PRINTER_DEVICE_NAME}_stop_printing
         card_mod:
           style: |
-              ha-card {
-                box-shadow: none !important;
-                background-color: rgba(0,0,0,0);
-                border: none;
-                --paper-item-icon-color: {% if is_state('sensor.{HA_PRINTER_DEVICE_NAME}_print_status', 'running') or is_state('sensor.{HA_PRINTER_DEVICE_NAME}_print_status', 'pause') %} red; {% else %} grey; {% endif %}            
-              }
-  - type: custom:mushroom-title-card
```

Comment:
This should have been moved several versions ago, but seems I had missed it for hacs version.